src/components/QrCode.vue

```diff
@@ -84,6 +84,12 @@
         })
         public size!: number;
 
+        public async toDataUrl(type: 'image/png') {
```

Comment:
Is the string 'image/png' supposed to be the default value? Because as it's right now it's the _type_ of the variable. Did you mean to write it with an equal sign?

Reply:
Ha, stupid mistake :D
Thanks